api/tests/api/helpers.rs

```diff
@@ -0,0 +1,29 @@
+use api::PostieApi;
+
+pub struct TestApp {
```

Comment:
I kind of based this testing pattern around what was introduced in Zero2Prod - having a main TestApp that has some reusable helper functions and have a place to stand up other test clients (like db), and perform any other initialization.